starter-code/index.html

```diff
@@ -0,0 +1,306 @@
+            <img src="../starter-code/img/slider-3.jpg" class="d-block w-100" alt="image of flowers">
+          </div>
+          <div class="carousel-item">
+            <img src="../starter-code/img/slider-4.jpg" class="d-block w-100" alt="image of people walking">
+          </div>
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleFade" role="button" data-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+          <span class="sr-only">Previous</span>
+        </a>
+        <a class="carousel-control-next" href="#carouselExampleFade" role="button" data-slide="next">
+          <span class="carousel-control-next-icon" aria-hidden="true"></span>
+          <span class="sr-only">Next</span>
+        </a>
+      </div>
+    </section>
+    <section>
+      <div class="row text-center">
+        <div class="col-lg-3 col-md-6 nb-4">
+          <div class="card" style="width: 18rem;">
+            <img src="../starter-code/img/cup.png" class="card-img-top" alt="cup">
+            <div class="card-body">
+              <h5>Web design</h5>
+              <p class="card-text">
+                iCum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum
+                vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 nb-4">
+          <div class="card" style="width: 18rem;">
+            <img src="../starter-code/img/star.png" class="card-img-top" alt="star">
+            <div class="card-body">
+              <h5>Web development</h5>
+              <p class="card-text">
+                iCum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum
+                vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 nb-4">
+          <div class="card" style="width: 18rem;">
+            <img src="../starter-code/img/heart.png" class="card-img-top" alt="heart">
+            <div class="card-body">
+              <h5>User interface</h5>
+              <p class="card-text">
+                iCum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum
+                vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="jumbotron">
+        <h4 class="display-4">LATEST PROJECTS</h>
+          <p class="lead">
+            FEW OF THE LATEST PORTFOLIO ITEMS
+          </p>
+          <hr class="my-4">
+          <div class="row text-center">
+            <div class="col-lg-3 col-md-6 nb-4">
```

Comment:
Cuidado com copy/paste
```suggestion
            <div class="col-lg-3 col-md-6 mb-4">
```